models/standardized_models/hubspot__marketing_campaigns_standardized.sql

```diff
@@ -0,0 +1,34 @@
+{{ config(enabled=(var('hubspot__standardized_marketing_model_enabled', False)
+        and fivetran_utils.enabled_vars(['hubspot_marketing_enabled','hubspot_email_event_enabled','hubspot_email_event_sent_enabled']))) 
+}}
+
+with campaigns as (
+
+    select *
+    from {{ ref('hubspot__email_campaigns') }} 
+),
+
+standardized as (
+
+    select 
+        email_campaign_id as campaign_id,
+        email_campaign_name as campaign_name,
+        email_campaign_type as campaign_type,
+        email_campaign_sub_type as campaign_sub_type,
+        email_campaign_subject as campaign_subject,
+        coalesce(app_id, content_id) as secondary_id,
+        cast(null as {{ dbt.type_timestamp() }}) as created_at,
+        cast(null as {{ dbt.type_timestamp() }}) as updated_at,
+        total_bounces,
+        total_clicks,
+        total_deferrals,
+        total_deliveries,
+        total_opens,
+        cast(null as {{ dbt.type_int() }}) as total_unsubscribes,
```

Comment:
Is this not available? It looks like we should have the information to generate unsubscribes from the reference table.

![image](https://github.com/user-attachments/assets/6d46dd0f-7a61-49d9-a23f-826f01f234c6)

Reply:
They were available, but took some rejiggering of the models to make this work. You can see the updated CHANGELOG for details (will add more in the Height ticket).